lib/tasks/end_users_switch.rake

```diff
@@ -0,0 +1,11 @@
+
+namespace :end_user_switch do
```

Comment:
I've checked `CodeClimate` and it complains in 2 issues:
![image](https://user-images.githubusercontent.com/11318903/61730138-0cdd7980-ad79-11e9-81b9-eda243614134.png)

The last one I don't mind if ops agrees (as said in https://github.com/3scale/porta/pull/1012/files#r306419635), but the first one is very easy to fix 😄 just running locally `rubocop lib/tasks/end_users_switch.rake -a` it fixes it alone, otherwise is adding manually that line on the top of the file 😄